integration_tests/src/main/python/map_test.py

```diff
@@ -766,21 +765,53 @@ def do_it(spark):
                     'map_zip_with(a, b,  (key, value1, value2) -> key) as indexed',]
         value_type = data_gen.data_type.valueType
         if isinstance(value_type, IntegralType):
-            columns.extend(['a', 'b',
+            columns.extend([
                     'map_zip_with(a, b,  (key, value1, value2) -> coalesce(value1, 0) + coalesce(value2, 0)) as add',
                     'map_zip_with(a, b,  (key, value1, value2) -> coalesce(value1, 0) * coalesce(value2, 0)) as mul',
                     'map_zip_with(a, b,  (key, value1, value2) -> coalesce(value1, 0) - coalesce(value2, 0)) as sub',
                     'map_zip_with(a, b,  (key, value1, value2) -> coalesce(value1, 1) / coalesce(value2, 1)) as div',])
         if isinstance(value_type, StringType):
-            columns.extend(['a', 'b',
+            columns.extend([
                     'map_zip_with(a, b,  (key, value1, value2) -> concat(coalesce(value1, ""), "-test-", coalesce(value2, ""))) as string_concat',])
         if isinstance(value_type, ArrayType):
-            columns.extend(['a', 'b',
+            columns.extend([
                     'map_zip_with(a, b,  (key, value1, value2) -> concat(value1, value2)) as array_concat',])
         df = two_col_df(spark, data_gen, data_gen)
         return df.selectExpr(columns)
     assert_gpu_and_cpu_are_equal_collect(do_it)
 
+@pytest.mark.parametrize('data_gen', [MapGen(IntegerGen(False), ArrayGen(int_gen, max_length=5), max_length=5)], ids=idfn)
```

Comment:
Here is the same problem as the others. I explicitly said ByteGen on the holes that the 256 different possible keys might give us duplicates, but I think that is too large also. I would ranther have 10 or 20 unique keys and unique values for this one.